helper/hex/hex.go

```diff
@@ -45,13 +45,13 @@ func DecodeHex(str string) ([]byte, error) {
 }
 
 // MustDecodeHex type-checks and converts a hex string to a byte array
-func MustDecodeHex(str string) []byte {
+func MustDecodeHex(str string) ([]byte, error) {
```

Comment:
Could we rename it to `DecodeHex`?
The `Must` would panic, it is a golang convention.

Reply:
keep with edge.

Reply:
replace MustDecodeHex  with DecodeHex